include/oneapi/dpl/pstl/parallel_impl.h

```diff
@@ -39,34 +39,37 @@ __parallel_find(__parallel_tag<_IsVector>, _ExecutionPolicy&& __exec, _Index __f
 {
     using __backend_tag = typename __parallel_tag<_IsVector>::__backend_tag;
 
-    typedef typename ::std::iterator_traits<_Index>::difference_type _DifferenceType;
-    const _DifferenceType __n = __last - __first;
-    _DifferenceType __initial_dist = _IsFirst::value ? __n : -1;
+    typedef typename std::iterator_traits<_Index>::difference_type _DifferenceType;
 
-    constexpr auto __comp = ::std::conditional_t<_IsFirst::value, __pstl_less, __pstl_greater>{};
+    return __internal::__except_handler([&]() {
```

Comment:
This change isn't required because `__internal::__except_handler` used on caller side of `__parallel_find` function.